### ISSUE TITLE
Postman app shortcut pinned on start menu on Windows 10 hasn't a thumbnail

### ISSUE THREAD

**Issue by a-marinin** [a85/POSTMan-Chrome-Extension#3752]
## App Details:

```
Postman app for Windows (ver. 5.3.2)
Windows 10 Pro (ver. 1607, OS Build 14393.1770)
```

## Issue Report:
**Description**: Postman app shortcut pinned on start menu on Windows 10 hasn't a thumbnail.
**Environment**: Windows 10 Pro
**Steps to Reproduce**:

1. Uninstall Postman app for Windows;
2. On Windows, click on the "start" icon;
3. Write down "Postman" on the search field;
4. Right click on "Postman Desktop app"
5. Click on "Pin to start" option;
6. Close the start menu;
7. Click on the "start" icon one more time;
8. On the start menu, review pinned Postman shortcut.

**Actual Results**:
The pinned Postman shortcut hasn't a thumbnail _(It's just black)_.

**Expected Results**:
The pinned Postman shortcut should have a thumbnail _(For example, a Postman icon)_.

**Attachments**
![postman app pinned to start menu](https://user-images.githubusercontent.com/33235310/32571912-cc0f4df4-c4a7-11e7-8fb1-a7fe27531708.png)

**Comment by prashantagarwal** [comment]
@a-marinin Thanks for pointing this out. We will get this fixed.

**Comment by SamvelRaja** [comment]
@a-marinin  this issue had been fixed in our 6.1.4 version of our app, kindly update it. If you still faces the issue kindly follow the steps from this comment https://github.com/postmanlabs/postman-app-support/issues/3107#issuecomment-307015494

**Comment by a-marinin** [comment]
@SamvelRaja sounds good. Thank you guys!
